core/modules/system/system.theme.inc

```diff
@@ -798,3 +798,35 @@ function system_preprocess_page_components(&$variables) {
     'page_components__' . $child_delta,
   );
 }
+
+/**
+ * Returns HTML for the display of a label with both human and machine names.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - label: The human-readable name or label.
+ *   - link: (optional) An associative array containing:
+ *     - path: The path to link the label to.
+ *     - options: (optional) An associative array of additional options used by
+ *       the l() function.
+ *   - machine_name: (optional) The machine-safe name or label.
```

Comment:
We shouldn't be confusing the use of word "label", which we usually reserve for non-machine names.

```suggestion
 *   - machine_name: (optional) The machine-safe name.
```